src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1160,6 +1231,24 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         scrollBar.Value(viewTop);
     }
 
+    // Method Description:
+    // - Called continuously to smoothly scroll viewport when user is
+    //      mouse selecting outside it (to 'follow' the cursor).
+    // Arguments:
+    // - none
+    void TermControl::_UpdateAutoScroll(Windows::Foundation::IInspectable const& /* sender */,
+                                        Windows::Foundation::IInspectable const& /* e */)
+    {
+        if (_autoScrollVelocity != 0)
+        {
+            _scrollBar.Value(_scrollBar.Value() + _autoScrollVelocity * _AutoScrollTimerInterval);

```

Comment:
I just want to say. I understand this because of Physics class and I never thought that moment would come.

Reply:
Just everyday snippet at gamedev :)